DietPi-Banner: Display total space and space used in % for the freespace option

Andr3Carvalh0 opened the pull request:
Simple change... Instead of only showing the available space for rootfs/userdata also indicate the total amount and the percentage thats in use.

<img width="682" alt="Screenshot 2024-01-05 at 13 31 19" src="https://github.com/MichaIng/DietPi/assets/6016433/b0d9c128-704e-445d-bf85-28292b711d9f">

Andr3Carvalh0 committed:
DietPi-Banner: Display total space and used % when freespace is displayed

MichaIng commented on dietpi/func/dietpi-banner:
```diff
@@ -248,9 +248,9 @@ $GREEN_LINE"
 		(( ${aENABLED[13]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[13]} $GREEN_SEPARATOR $(/boot/dietpi/dietpi-vpn status 2>&1)"
 		# Freespace (RootFS)
-		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h --output=avail / | mawk 'NR==2 {print $1}' 2>&1)"
+		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h / | mawk 'NR==2 {printf "%s / %s (%s)", $4, $2, $5}' 2>&1)"
```
Not really shorter or so, but I like to be explicit about the processed fields:
```suggestion
		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h --output=avail,size,pcent / | mawk 'NR==2 {printf $1" / "$2" ("$3"%)"}' 2>&1)"
```

Andr3Carvalh0 replied:
Good idea!
Ive updated the PR + in the end I started to think that showing the used percentage might be a bit confusing, since you have displaying the free space / total space, so I ended up removing it...

MichaIng commented on dietpi/func/dietpi-banner:
```diff
@@ -248,9 +248,9 @@ $GREEN_LINE"
+		(( ${aENABLED[7]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[7]} $GREEN_SEPARATOR $(df -h / | mawk 'NR==2 {printf "%s / %s (%s)", $4, $2, $5}' 2>&1)"
 		# Freespace (DietPi userdata)
-		(( ${aENABLED[8]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[8]} $GREEN_SEPARATOR $(df -h --output=avail /mnt/dietpi_userdata | mawk 'NR==2 {print $1}' 2>&1)"
+		(( ${aENABLED[8]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[8]} $GREEN_SEPARATOR $(df -h /mnt/dietpi_userdata | mawk 'NR==2 {printf "%s / %s (%s)", $4, $2, $5}' 2>&1)"
```
```suggestion
		(( ${aENABLED[8]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[8]} $GREEN_SEPARATOR $(df -h --output=avail,size,pcent /mnt/dietpi_userdata | mawk 'NR==2 {printf $1" / "$2" ("$3"%)"}' 2>&1)"
```

Andr3Carvalh0 committed:
Make processed fields more explicit

Andr3Carvalh0 committed:
Rename Freespace to Disk usage, and re-enable used percentage

MichaIng committed:
v9.0

- CHANGELOG | DietPi-Banner: Instead of "Freespace", the "Disk usage" is now shown, including the total disk size and used percent. Many thanks to @Andr3Carvalh0 for implementing this change: https://github.com/MichaIng/DietPi/pull/6837
